docs/_plugins/search_index.rb

```diff
@@ -192,10 +198,11 @@ def execute
     edition_short: edition_short,
     date: post.data['date'].strftime('%F'),
     supported: !!post.data['supported'],
-    body: body
+    body: body,
+    url: post.url
   }
 
-  uniq = "#{post.data['name']}_#{post.data['language']}"
+  uniq = "#{post.data['name']}_#{post.data['edition']}_#{post.data['language']}"
```

Comment:
You break the purpose of this variable. It needs for collecting model editions. You need to create another variable with the same value.

Reply:
Agreed .. will change.